### PR TITLE
Update on entity docs

### PR DESCRIPTION
**Pull request by 12AwsomeMan34**

### CODE REVIEW

File: source/plugin/basics/entities/spawning.rst [hunk 1]
```diff
@@ -11,22 +11,24 @@ For example, let's try to spawn a Creeper:
...
+        Optional<Entity> optional = extent.createEntity(EntityTypes.CREEPER, location.getPosition());
         if (optional.isPresent()) {
             // After this, we can use more API that relates to creeper
             Creeper creeper = (Creeper) optional.get();
-            creeper.setPowered(true);
-            creeper.setExplosionRadius(10);
+            ExplosiveRadiusData radiusData = creeper.getExplosiveRadiusData();
+            radiusData.setExplosionRadius(10);
+            // Return the modified explosive radius data back to the creeper
+            creeper.offer(radiusData);
```
Comment (gabizou) [inline source/plugin/basics/entities/spawning.rst]: Why not add `ChargedData` and `DisplayNameData` to show a bit more of an example?

Comment (gabizou) [inline source/plugin/basics/entities/spawning.rst]: Also, given that this is all `Data API`, we don't even need to cast anymore to `Creeper`.

Comment (gabizou) [inline source/plugin/basics/entities/spawning.rst]: @12AwsomeMan34 pinging for changes requested.

Comment (12AwsomeMan34) [inline source/plugin/basics/entities/spawning.rst]: Will get to doing this after work.

**Pull request merged**